### ISSUE TITLE
GCHP simulation times out after read_prefetch step, caused by incorrect variable name in ExtData.rc

### ISSUE THREAD

**Issue by yantosca** [geoschem/gchp#373]
I've been working on adding the carbon simulation to GCHP and I've come across an error where MAPL seems to hang at the read_prefetch step when encountering an invalid variable name.

In the allPEs.log file:
```console
95043:0001: CAP.EXTDATA: DEBUG:    ExtData Run_: DO_UPDATE: Start. doUpdate_ is true.
195055:0001: CAP.EXTDATA: DEBUG:    ExtData Run_: Has_Run: Start. hasRun is false. Update time.
195067:0001: CAP.EXTDATA: DEBUG:       ExtData Run_: HAS_RUN: NotSingle is false. Just get time on file.
195079:0001: CAP.EXTDATA: DEBUG:  Retrieving formatter for: ./HcoDir/MASKS/v2018-09/Mexico_Mask_Mirror.001x001.nc
195091:0001: CAP.EXTDATA: DEBUG:       ExtData Run_: HAS_RUN: End
195103:0001: CAP.EXTDATA: DEBUG:    ExtData Run_: DO_UPDATE: End
195115:0001: CAP.EXTDATA: DEBUG: ExtData Run_: READ_LOOP: Done
195127:0001: CAP.EXTDATA: DEBUG: ExtData Run_: ---PopulateBundle
195139:0001: CAP.EXTDATA: DEBUG: ExtData Run_: ---prefetch
195153:0001: CAP.EXTDATA: DEBUG: ExtData Run_: ---IclientDone
```

In the slurm output file:
```console
+ srun -n 12 -N 1 -m plane=12 --mpi=pmix ./gchp
pe=00009 FAIL at line=00029    NetCDF4_get_var.H                        <Variable not found: emi_ch4_1B2b_Natural_Gas_Transmission in file: ./HcoDir/CH4/v2023-07/Gridded_GHGI_v2/Express_Extension_Gridded_GHGI_Methane_v2_2019.nc>
pe=00009 FAIL at line=00705    ServerThread.F90                         <status=1>
pe=00009 FAIL at line=00469    ServerThread.F90                         <status=1>
pe=00009 FAIL at line=01096    ServerThread.F90                         <status=1>
pe=00009 FAIL at line=00088    MessageVisitor.F90                       <status=1>
pe=00009 FAIL at line=00113    AbstractMessage.F90                      <status=1>
pe=00009 FAIL at line=00107    SimpleSocket.F90                         <status=1>
pe=00009 FAIL at line=00407    ClientThread.F90                         <status=1>
pe=00010 FAIL at line=00029    NetCDF4_get_var.H
... etc...
srun: Job step aborted: Waiting up to 32 seconds for job step to finish.
slurmstepd: error: *** JOB 15539922 ON holy8a29609 CANCELLED AT 2024-01-12T19:40:13 DUE TO TIME LIMIT ***
slurmstepd: error: *** STEP 15539922.0 ON holy8a29609 CANCELLED AT 2024-01-12T19:40:13 DUE TO TIME LIMIT ***
```
Instead of exiting with an error the job hangs until timeout is encountered.

The cause of this error was this entry in `ExtData.rc`:
```console
GHGI_EE_GAS_TRANSMISSION        molec/cm2/s N    Y - none none  emi_ch4_1B2b_Natural_Gas_Transmission ./HcoDir/CH4/v2023-07/Gridded_GHGI_v2/Express_Extension_Gridded_GHGI_Methane_v2_%y4.nc
```
which should have been:
```console
GHGI_EE_GAS_TRANSMISSION        molec/cm2/s N    Y - none none  emi_ch4_1B2b_Natural_Gas_TransmissionStorage ./HcoDir/CH4/v2023-07/Gridded_GHGI_v2/Express_Extension_Gridded_GHGI_Methane_v2_%y4.nc
```
Files:
[slurm-15539922.out.txt](https://github.com/geoschem/MAPL/files/13952996/slurm-15539922.out.txt)
[gchp.20190101_0000z.txt.log](https://github.com/geoschem/MAPL/files/13952998/gchp.20190101_0000z.txt.log)
[allPEs.log.txt.log](https://github.com/geoschem/MAPL/files/13953014/allPEs.log.txt.log)
[cap_restart.txt](https://github.com/geoschem/MAPL/files/13953017/cap_restart.txt)
[logfile.000000.out.txt.txt](https://github.com/geoschem/MAPL/files/13953012/logfile.000000.out.txt.txt)

Tagging @lizziel

**Comment by lizziel** [comment]
I am going to transfer this to GCHP GitHub so users can see it and it can be searchable there.

**Comment by lizziel** [comment]
I created an issue on the upstream MAPL repository here: https://github.com/GEOS-ESM/MAPL/issues/2616

**Comment by lizziel** [comment]
I am closing this issue since we anticipate a fix in MAPL 3 which will be included in GCHP 15.0.
